compile/compile_test.go

```diff
@@ -2638,6 +2654,7 @@ q[3]
 			for _, useMemoryFS := range []bool{false, true} {
 				test.WithTestFS(tc.modules, useMemoryFS, func(root string, fsys fs.FS) {
 					compiler := New().
+						WithRegoVersion(ast.RegoV1).
```

Comment:
I imagine there's a reason for explicitly specifying the rego version and not just use the default. There are more instances of the same in the other places. So that's probably on purpose but please check before merging.